src/user-research/choosing-and-using-research-methods.md

```diff
@@ -1,13 +1,15 @@
 ---
 title: Choosing and using research methods
 related_order: 70
-last_reviewed_at: ""
+last_reviewed_at: 2024-01-29T11:17:41.859Z
 ---
 Our fourth research principle is to [be methodical, but not rigid](/user-research/#user-research-principles). At dxw, we don’t have a fixed set of approved research methods. But when we [plan our research](https://playbook.dxw.com/user-research/creating-and-using-research-plans/), we do need to choose appropriate methods for the context and apply them well.
 
 We prefer tried and trusted methods that can provide strong evidence and reliable answers to our questions, for the least time, effort and cost.
 
-And we always consider a team’s level of experience with user research. So we may prefer simpler methods like interviews and usability testing that the team can easily understand and join in with.
+And we make sure to use these methods in ways that are [inclusive and accessible](/user-research/making-research-activities-inclusive-and-accessible/), and [safe for our participants and ourselves](/user-research/doing-research-safely/).
+
+We also consider a team’s level of experience with user research. So we may prefer simpler methods like interviews and usability testing that the team can easily understand and join in with.
```

Comment:
join in with. The preposition 'with' reads redundant after 'join in'.

Reply:
Replace with "get involved in".